helper_test.go

```diff
@@ -60,7 +60,7 @@ func createAgentPool(t *testing.T, client *Client, org *Organization) (*AgentPoo
 
 	return pool, func() {
 		if err := client.AgentPools.Delete(ctx, pool.ID); err != nil {
-			t.Errorf("Error destroying agent pool! WARNING: Dangling resources "+
```

Comment:
I think we should do the same thing on org cleanup. I think this error may prevent org cleanup until org delete is async

Reply:
OK - is there a way we could confirm that?

Reply:
No, aside from seeing if we hit any new dangling resource errors without it.